src/h2olog/h2olog.cc

```diff
@@ -190,12 +229,13 @@ int main(int argc, char **argv)
     }
 
     int debug = 0;
+    int drop_root = 1;
```

Comment:
I think it might be less confusing if we rename this to `preserve_root` or something to avoid using inverted definitions against the command line option.

Reply:
Nice! yes, thank you for pointing this out. This definitely requires few brain cycles to grok.